apisix/plugins/kafka-logger.lua

```diff
@@ -205,6 +201,11 @@ function _M.log(conf, ctx)
         end
     end
 
+    if core.table.nkeys(broker_list) == 0 then
```

Comment:
Could we validate it via JSONSchema?

Reply:
Validate it via JSONSchema will rewrite the schema definition(broker_list):
![image](https://user-images.githubusercontent.com/75366457/130381824-5864f96f-873d-428d-b957-820cc3f8931d.png)

Reply:
Yes. But as we already check it elsewhere, check it in JSONSchema doesn't break.

Reply:
okay~

Reply:
Could you provide some examples?

Reply:
Currently the `broker_list` is `object` type, if redefine the type of `broker_list` will break(e.g. `array` type), so in this case, would you give some suggestion~

Reply:
We don't need to redefine the type to "array". We can add validation to the object type. See https://json-schema.org/understanding-json-schema/reference/object.html

Reply:
the document has been seen, and we can add `minProperties = 1` validation to the `broker_list` schema, but this logic `if core.table.nkeys(broker_list) == 0 then` is also required, because check `port` type is number.

Reply:
We can check port type too: https://json-schema.org/understanding-json-schema/reference/object.html#pattern-properties
You can find some examples in APISIX.